alexandria/settings/alexandria.py

```diff
@@ -194,3 +194,15 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
     "MANABI_SHARED_KEY", default=default("2QhWg20fXq0xlnJUkFTDgFoA3JWqvb86OejD9mAVFCW")
 )
 MANABI_SECURE = True if ENV == "production" else False
+
+
+# Content search
+ALEXANDRIA_ENABLE_CONTENT_SEARCH = env.bool(
```

Comment:
Should this feature be configurable to be on or off?

Reply:
I think it should be default-off, as it requires additional infrastructure and CPU load